src/vehicle.cpp

```diff
@@ -3531,16 +3531,25 @@ void vehicle::slow_leak()
 
         auto fuel = p.ammo_current();
         int qty = std::max( ( 0.5 - health ) * ( 0.5 - health ) * p.ammo_remaining() / 10, 1.0 );
+        point q = coord_translate( p.mount );
+        const tripoint dest = global_pos3() + tripoint( q.x, q.y, 0 );
 
-        // damaged batteries self-discharge without leaking
-        if( fuel != fuel_type_battery ) {
+        // damaged batteries self-discharge without leaking, plutonium leaks slurry
+        if( fuel != fuel_type_battery && fuel != "plut_cell" ) {
```

Comment:
You can make add `fuel_type_plutonium_cell` and use it instead of `"plut_cell"`:

```CPP
static const itype_id fuel_type_plutonium_cell ( "plut_cell" );
```